pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -551,6 +551,9 @@ const (
 	LabelProxy = "proxy"
 	// LabelExtensionProjectRole is a constant for a label value for extension project roles
 	LabelExtensionProjectRole = "extension-project-role"
+	// LabelExtensionWebhookControlplaneSelector is a constant for a label key for webhook object filter targets
+	//  the value can be true or false
+	LabelExtensionWebhookControlplaneSelector = LabelExtensionPrefix + "webhook-controlplane-selector"
```

Comment:
I'm open for label naming suggestions. Label proposal: `extensions.gardener.cloud/webhook-controlplane-selector=true`